appendices/spirvenv.txt

```diff
@@ -717,7 +717,7 @@ endif::VK_VERSION_1_2,VK_KHR_shader_float16_int8[]
   ** *Private*
   ** *Function*
 ifdef::VK_KHR_shader_clock[]
-  * Scope for code:OpReadClockKHR must be limited to:
+  * Scope for code:OpReadClockKHR must: be limited to:
```

Comment:
Any edits to normative terms, while almost certainly correct, will still require increased scrutiny by the WG before it will sign off on them. So I suggest splitting these changes out. Things that are actually just typos / markup errors the WG will usually just approve on my say-so so they're faster.

Reply:
That shouldn't be a problem.

I edited only the ones that I were 110 % sure about and only the tip of the iceberg. There seems to be a large number of colloquial use of the words rather than the RFC meaning. :/

Reply:
@oddhack I think this PR should be safe now.